packages/ini/test.pony

```diff
@@ -26,7 +28,9 @@ class iso _TestIniParse is UnitTest
     let array: Array[String] = source.split("\r\n")
     let map = IniParse(array.values())
 
+    h.assert_eq[Bool](map.contains("Section 2"), true)
+
```

Comment:
any reason for this to be hanging out by itself with this line dividing it from its brethren?